components/imageproc/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "imageproc"
+version = "0.1.0"
+authors = ["Vojtěch Král <vojtech@kral.hk>"]
+
+[dependencies]
+lazy_static = "1"
+regex = "0.2"
+tera = "0.11.0"
+image = "0.18.0"
+rayon = "0.9"
+twox-hash = "1.1"
```

Comment:
Note to myself: this crate seems to depend on an old rand version, might be worth doing a PR to upgrade it